src/segyviewlib/version.py

```diff
@@ -1 +1 @@
-__version__ = '1.0.6'
+version = '0.0.0'
```

Comment:
it's not as big setback as it seems. 

setup.py uses setuptools.scm to overwrite the version number and is based on the Git tag. The version.py should likely be in .gitignore.